pages/about/code-of-conduct.md

```diff
@@ -34,3 +34,13 @@ $(document).ready(function(){
 
 });
 </script>
+
+<p>
+    The current Code of Conduct Committee consists of:
+    <ul>
+        <li> Lezlie Espana</li>
+        <li> David Nicholson</li>
+        <li> Joshua Teves</li>
+    </ul>
+    You may reach any of them on the US-RSE slack individually or email [coc@us-rse](mailto:contact@us-rse.org) in order to contact them.
```

Comment:
0. The mailto: email address (display text) is missing ".org" and doesn't match the actual mailto address.
1. The mailto: link isn't [rendering properly in the preview](https://output.circle-artifacts.com/output/job/bafb1d75-ffbd-4a0f-ac9c-df6d87ed8109/artifacts/0/usrse.github.io/about/code-of-conduct/index.html) for some reason.
2. I would suggest capitalizing "Slack" ~and consider adding a link to the Slack directly~ (removed about adding link to Slack, doesn't seem that it's done throughout the website).